CONTRIBUTING.md

```diff
@@ -130,11 +350,17 @@ make -f Makefile.docker build
 
 ### Running the integration tests
 
+> NOTE: Some parts of the integration tests are not configurable and therefore
+> cannot be run by folks outside the core team. If you are NOT contributing to the
+> gitops functionality, you can run a subset of the tests which cover your change,
+> see below.
+
 The integration tests are long and unfortunately there are some flakes (help is very
 welcome!).
 
```

Comment:
shall we note that it might be costly for them to run? I have a feeling each run probably costs a fair few $$ since AWS charges by the hour

Reply:
Yep good idea